src/terminal/terminal.c

```diff
@@ -1584,8 +1587,8 @@ static int __guac_terminal_send_key(guac_terminal* term, int keysym, int pressed
     /* If key pressed */
     else if (pressed) {
 
-        /* Ctrl+Shift+V shortcut for paste */
-        if (keysym == 'V' && term->mod_ctrl)
+        /* Ctrl+Shift+V or Cmd+v (mac style) shortcuts for paste */
+        if ((keysym == 'V' && term->mod_ctrl) || (keysym = 'v' && term->mod_meta))
```

Comment:
This an assignment of `'v'` to `keysym`. It should instead be `keysym == 'v'`.

Reply:
Oops, that would explain the weird behavior I saw when retesting this. Fixed!